### PR TITLE
Fix #193: Surface configuration options

### LINKED ISSUE #193: Allow users to more easily select populations
Currently, the user can visit the configuration page to change the populations that are displayed. I talked with Rob Miller about making that even easier, for example having some type of multi-select widget above the legend. Davor has also suggested having a lighbox overlay, which I also think would work well.

One added benefit of this is that it would allow users to more easily change the populations that are shown on the chart detail page. (At the moment, the configuration page always returns the user to the homepage chart listing even if they came from a chart detail page.)

### PR DESCRIPTION
**Pull request by openjck**
This change also makes the "Show outliers" option available on the
homepage and the "Populations" option available on the chart detail
page.

### CODE REVIEW

File: distributionviewer/core/static/css/configuration.styl [hunk 1]
```diff
@@ -0,0 +1,87 @@
+@import 'lib';
+
+.configuration {
+  margin-bottom: $med-padding;
+  max-height: 200px;
+  overflow-y: auto;
+  padding: $min-padding;
+
+  &[open] {
+    margin-bottom: $lrg-padding;
```
Comment (spasovski) [inline distributionviewer/core/static/css/configuration.styl:10]: Can probably move this to the same target below.

File: distributionviewer/core/static/js/app/components/containers/configuration-container.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    if (props.configurePopulations) {
+      const populations = new Populations();
+      this.populationObjects = populations.getPopulations();
+    }
+
+    this._updateQueryParameter = this._updateQueryParameter.bind(this);
+    this._handleModifyOutliers = this._handleModifyOutliers.bind(this);
+    this._handleModifyScale = this._handleModifyScale.bind(this);
+    this._handleModifyPopulations = this._handleModifyPopulations.bind(this);
+    this._handleModifyCharts = this._handleModifyCharts.bind(this);
```
Comment (spasovski) [inline distributionviewer/core/static/js/app/components/containers/configuration-container.js:26]: `this` is the cost of not using `React.createClass()` 😁

**Comment by spasovski** [comment]
I miss the days when this project was simple. r+

File: distributionviewer/core/static/css/configuration.styl (hunk 1 above)
Comment (openjck) [reply distributionviewer/core/static/css/configuration.styl:10]: I'll keep this here just to keep all the visibility stuff together. The compressor should combine them, thankfully.

**Pull request merged**